doc/devices/dts/device_tree.rst

```diff
@@ -80,6 +80,59 @@ This .fixup file by default resides in the board directory and is named
 dts.fixup.  This fixup file maps the generated include information to the
 current driver/source usage.
 
+.. _dt_vs_kconfig:
+
+Device Tree vs Kconfig
+**********************
+
+As mentioned above there are several tools used to configure the build with
+Zephyr.
+The two main ones, Device Tree and Kconfig, might seem to overlap in purpose,
+but in fact they do not. This section inteds to serve as a reference when
```

Comment:
Change to: (and removing the misspelled "intends")

    This section serves as a reference to help you decide whether to place
    configuration items in Device Tree or Kconfig.

Reply:
fixed